modules/cloud-run/main.tf

```diff
@@ -75,6 +72,9 @@ locals {
     var.gen2_execution_environment ? {
       "run.googleapis.com/execution-environment" = "gen2"
     } : {},
+    var.startup_cpu_boost ? {
+      "run.googleapis.com/startup-cpu-boost" = "true"
+    } : {},
```

Comment:
according to https://cloud.google.com/run/docs/reference/rest/v1/ObjectMeta must be in revision